manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  "vendor": "CHANGE_ME",
-  "name": "CHANGE_ME",
-  "version": "0.0.0",
-  "title": "CHANGE_ME",
-  "description": "CHANGE_ME",
+  "vendor": "vtex",
```

Comment:
The vendor needs to be `vtexbr` now, doesn't it?

Reply:
You are 100% right